### PR TITLE
Added jss-preset-default

### PR DESCRIPTION
**Pull request by HenriBeck**
Do not merge yet. The build is still broken.

### CODE REVIEW

File: packages/jss-preset-default/src/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import camelCase from 'jss-plugin-syntax-camel-case'
+import defaultUnit from 'jss-plugin-syntax-default-unit'
+import expand from 'jss-plugin-syntax-expand'
+import vendorPrefixer from 'jss-plugin-vendor-prefixer'
+import propsSort from 'jss-plugin-props-sort'
+
+export default (options = {}) => ({
+  plugins: [
+    functionPlugin(options.function),
+    observablePlugin(options.observable),
```
[the remark this answers is not in the document]
Comment (HenriBeck) [reply packages/jss-preset-default/src/index.js]: we could remove the plugin from the observable but the functionPlugin is sadly required.

[the remark this answers is not in the document]
Comment (kof) [reply packages/jss-preset-default/src/index.js]: call it functions :)

**Comment by HenriBeck** [comment]
The weird things, whenever rollup tries to bundle a file with the `_typeof` helper from babel it fails...

That's the reason why the build is currently failing.

File: packages/jss-preset-default/src/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (kof) [reply packages/jss-preset-default/src/index.js]: or add Plugin suffix everywhere)

**Comment by kof** [comment]
hmm, thats weird

**Comment by kof** [comment]
mb @TrySound knows this problem

**Comment by HenriBeck** [comment]
I think the problem is that we don't transpile anything for the es modules then which would require the user to compile it. We should at least remove the flow types.

**Comment by TrySound** [comment]
I would suggest a simpler solution. Just generate cjs and esm bundles. Our distribution will be more maintainable and will be tracked with size snapshots.

You can see an example how I do this in powerplug. Also you probably will need to exclude paths started with \0 too.

**Pull request merged**